cmd/get_database.go

```diff
@@ -1,31 +1,239 @@
 package cmd
 
 import (
-	"errors"
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
 
 	"github.com/YaleSpinup/spinup-cli/pkg/spinup"
-	"github.com/spf13/cobra"
 )
 
-/* Initialized the command in the getCmd thing. */
-func init() {
-	getCmd.AddCommand(getDatabaseCmd)
-}
+func getDatabase(params map[string]string, resource *spinup.Resource) error {
+	var j []byte
+	var err error
+	status := resource.Status
 
-/* Defines the cmd via a bunch of flags
-Using the cobra library */
-var getDatabaseCmd = &cobra.Command{
-	Use:   "database",
-	Short: "Get details about a database resource",
-	RunE: func(cmd *cobra.Command, args []string) error {
-		if len(args) != 1 {
-			return errors.New("exactly 1 container service id is required")
-		}
-		resource := &spinup.Resource{}
-		if err := SpinupClient.GetResource(map[string]string{"id": args[0]}, resource); err != nil {
+	if status != "created" && status != "creating" && status != "deleting" {
+		j, err = ingStatus(resource)
+		if err != nil {
 			return err
 		}
+	} else {
+		if detailedGetCmd {
+			if j, err = databaseDetails(params, resource); err != nil {
+				return err
+			}
+		} else {
+			if j, err = database(params, resource); err != nil {
+				return err
+			}
+		}
+	}
+
+	f := bufio.NewWriter(os.Stdout)
+	defer f.Flush()
+	f.Write(j)
+
+	return nil
+}
+
+func database(params map[string]string, resource *spinup.Resource) ([]byte, error) {
+	size, err := SpinupClient.DatabaseSize(resource.SizeID.String())
+	if err != nil {
+		return []byte{}, err
+	}
+
+	info := &spinup.DatabaseInfo{}
+	if err := SpinupClient.GetResource(params, info); err != nil {
+		return []byte{}, err
+	}
+
+	status := resource.Status
+	if len(info.DBClusters) > 0 {
+		status = info.DBClusters[0].Status
+		if info.DBClusters[0].EngineMode == "serverless" && info.DBClusters[0].Capacity == 0 {
+			status = "paused"
```

Comment:
Now I see why it may be better handle this stuff in the API, instead of leaving it to the client/UI